lib/scaffold.dart

```diff
@@ -571,8 +571,11 @@ class BackdropScaffoldState extends State<BackdropScaffold>
               children: <Widget>[
                 Flexible(
                   child: _MeasureSize(
-                    onChange: (size) =>
-                        setState(() => _backPanelHeight = size.height),
+                    onChange: (size) {
+                      if (size != null) {
+                        setState(() => _backPanelHeight = size.height);
+                      }
+                    },
```

Comment:
Added enclosing `size != null` check. Is this ok?

Reply:
Resolved in https://github.com/fluttercommunity/backdrop/commit/2f4010e5619ad2089290835c3e40b7048594943f.